### DISCUSSION TITLE
Missing containers data in the Mem / MAX / IOrs / IOw rows

### DISCUSSION THREAD

**Discussion by MISIKEX** [nicolargo/glances#2921] [General]
Hello,

I have a troubble about i get “?” In the container details rows.

In he Memory, MAX, IOR/s, and IOW/s rows.

![IMG_0006](https://github.com/user-attachments/assets/5619bf75-fead-4cfe-893b-ccd848d7eb2c)

I using this compose:

version: "3.9"
services:
  glances:
    image: nicolargo/glances:latest-full
    container_name: glances
    pid: host
    privileged: true
    ports:
      - 7300:61208
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - /etc/os-release:/etc/os-release:ro
    environment:
      - TZ=Europe/Budapest
      - GLANCES_OPT=-w
    restart: unless-stopped

Any idea how to fix this?

thank you!

**Comment by nicolargo** [comment]
Hi @MISIKEX ,

do you have the stats when your run *docker stats* ?

**Reply by MISIKEX** [reply]
Hello!

No. Its shows the same:

![image](https://github.com/user-attachments/assets/42f20f58-d9f1-4da1-b0f8-cf1a02a84de1)

Any ide whats going on?

**Comment by MISIKEX** [comment]
Hello!

After you said the docker stats, and the same problem, i found the solution!

sudo vi /boot/firmware/cmdline.txt

_then adding these commands to the existing line:_
cgroup_enable=cpuset cgroup_enable=memory cgroup_memory=1

Then sudo reboot.

After these steps, everything working well!

Thank you for the suggestions! :)
